examples/tools/generate.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <generate.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Generate data samples"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "There are two different types of data samples: (1) :ref:`phase space samples\n",
+    "<usage/tools/generate:Phase space samples>` and (2) :ref:`intensity-based\n",
+    "samples <usage/tools/generate:Intensity-based samples>` (which mimick the\n",
+    "intensity profile of the decay topology). In addition, you may want to\n",
+    "generate more data points in specific regions of phase space (e.g. in the\n",
+    "case of a sharp resonance). This is called :ref:`importance sampling\n",
+    "<usage/tools/generate:Importance sampling>`.\n",
```

Comment:
should be written a bit more precise. So something like this:
In case of present narrow states importance sampling is beneficial. Instead of creating an unweighted uniform phase space sample a weighted sample with more points in the peak regions are generated.

Reply:
Okay thanks, implemented in the importance section itself
https://github.com/ComPWA/pycompwa/pull/79/commits/bd3e48cd35c29baaceb365f7afc72fa9e4139abb